DragaliaAPI.Shared/Definitions/Enums/Materials.cs

```diff
@@ -549,6 +549,61 @@ public static class UpgradeMaterials
         { Materials.SucculentDragonfruit, 3500 }
     };
 
+    public readonly static Dictionary<int, int> dragonBuildupXp = new Dictionary<int, int>()
```

Comment:
These data structures are handy but I'm not sure the enums folder is the best place for them